eoscostarica/guide.eoscostarica.io
# feat(frontend): improve homepage

### Improve Homepage Style and Adding Hero Images

### What does this PR do?

- resolves #439 and #168 

### Steps to test
1. Go to the Homepage
2. See new changes

#### CheckList
- [x] Images passing by carousel with new images
- [x] Change styles to cards
